packages/docs/sdk/react-sdk/api/connectTownsWithBearerToken.mdx

```diff
@@ -16,7 +16,7 @@ import { connectTownsWithBearerToken } from '@towns-protocol/react-sdk'
 ```ts
 function connectTownsWithBearerToken(
   token: string,
-  config: Omit<SyncAgentConfig, 'context'>,
+  config: Omit<SyncAgentConfig, "context">,
```

Comment:
double quotes now?

Reply:
I think that under the hood the typescript compiler is using double quotes to stringify. We're using the TS compiler to get those signatures.